ACE/ace/Configuration.h

```diff
@@ -520,7 +521,8 @@ class ACE_Export ACE_Configuration_Win32Registry : public ACE_Configuration
    */
   static HKEY resolve_key (HKEY hKey,
                            const ACE_TCHAR* path,
-                           int create = 1);
+                           int create = 1,
```

Comment:
Can you maybe at the same time change create to a bool? From very very long ago we didn't use bool because not all compilers did support it, but now we should use it

Reply:
Only this one or all `create` arg in these two files?

Reply:
Please update all create arguments in these two files

Reply:
Updated